### PR TITLE
[fix]: [PL-27331]: Updated redis client to develop

### PR DESCRIPTION
**Pull request by piyush-harness**
## Describe your changes

Retries got activated for create consumer group Redis action with this version upgrade so service startup time was significantly increased due to 10 retires in 1500 ms interval , Each consumer group it was taking 15 seconds and we have 20+ consumer on average groups in services.
We had to reduce the number of retries and the interval between them to 3 and 500ms ,This is not the final number it's implemented for the sake of moving ahead.

## Checklist
- [ ] I've documented the changes in the PR description.
- [ ] I've tested this change either in PR or local environment.
- [ ] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>
  
- Feature build: `trigger feature-build`
- Immutable delegate `trigger publish-delegate`
</details>

<details>
  <summary>PR Check triggers</summary>
  
  You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ti0, ti1`
  
- Compile: `trigger compile`
- CodeFormat: `trigger codeformat`
- MessageMetadata: `trigger messagecheck`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- runDockerizationCheck: `trigger dockerizationcheck`
- runAuthorCheck: `trigger authorcheck`
- Checkstyle: `trigger checkstyle`
- PMD: `trigger pmd`
- TI-bootstrap: `trigger ti0`
- TI-bootstrap1: `trigger ti1`
- TI-bootstrap2: `trigger ti2`
- TI-bootstrap3: `trigger ti3`
- TI-bootstrap4: `trigger ti4`
- FunctionalTest1: `trigger ft1`
- FunctionalTest2: `trigger ft2`
- CodeBaseHash: `trigger codebasehash`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

- - -
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/36230)

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Passed_** 
 


            
 No issues Everything looks good
 [Details](https://app.embold.io/organization/gh/harness/repositories/7752dbb3099d2dba3b5d37f5931a5e4c/pull-request/36230)

**Comment by piyush-harness** [comment]
trigger buildnum

File: 953-events-api/src/main/java/io/harness/eventsframework/impl/redis/RedisUtils.java [hunk 1]
```diff
@@ -68,7 +68,8 @@ public RedissonClient getClient(RedisConfig redisConfig) {
       }
 
       // Default retry interval is 1500 milliseconds
```
Comment (abhinav-harness) [inline 953-events-api/src/main/java/io/harness/eventsframework/impl/redis/RedisUtils.java:70]: remove default timeout comment too.

File: 953-events-api/src/main/java/io/harness/eventsframework/impl/redis/RedisUtils.java [hunk 2]
```diff
@@ -68,7 +68,8 @@ public RedissonClient getClient(RedisConfig redisConfig) {
...
-      serverConfig.setRetryAttempts(10);
+      serverConfig.setRetryAttempts(3);
+      serverConfig.setRetryInterval(500);
```
Comment (gn-harness) [inline 953-events-api/src/main/java/io/harness/eventsframework/impl/redis/RedisUtils.java:72]: why this change?

**Review by gn-harness — APPROVED** [review]

**Comment by piyush-harness** [comment]
*[`953-events-api/src/main/java/io/harness/eventsframework/impl/redis/RedisUtils.java` line 70 at r2](https://reviewable.io/reviews/harness/harness-core/36230#-N94gUQWInniDt7D2Cqx-r2-70:-N95Wu-m8itYkjeBY-E5:b4om10b) ([raw file](https://github.com/harness/harness-core/blob/61f78b3f4f5bfbc30df9c56258f1c689bb14f654/953-events-api/src/main/java/io/harness/eventsframework/impl/redis/RedisUtils.java#L70)):*
<details><summary><i>Previously, abhinav-harness (Abhinav Singh) wrote…</i></summary><blockquote>

remove default timeout comment too. 
</blockquote></details>

Let's keep it for reference when we come up with right numbers

**Comment by piyush-harness** [comment]
*[`953-events-api/src/main/java/io/harness/eventsframework/impl/redis/RedisUtils.java` line 72 at r2](https://reviewable.io/reviews/harness/harness-core/36230#-N94gUQWInniDt7D2Cqx-r2-72:-N95Ww264xXbG4Vykl-A:b-896fix) ([raw file](https://github.com/harness/harness-core/blob/61f78b3f4f5bfbc30df9c56258f1c689bb14f654/953-events-api/src/main/java/io/harness/eventsframework/impl/redis/RedisUtils.java#L72)):*
<details><summary><i>Previously, gn-harness (Gaurav Nanda) wrote…</i></summary><blockquote>

why this change?
</blockquote></details>

Done.
